src/api/types/TypeRegistrySpec.js

```diff
@@ -1,35 +1,50 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2017, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
 define(['./TypeRegistry', './Type'], function (TypeRegistry, Type) {
     describe('The Type API', function () {
-        var typeregistry;
+        var typeRegistryInstance;
 
         beforeEach(function () {
-            typeregistry = new TypeRegistry ();
-            typeregistry.addType('testtype', {
-                name: 'Test Type',
+            typeRegistryInstance = new TypeRegistry ();
+            typeRegistryInstance.addType('testType', {
+                label: 'Test Type',
```

Comment:
'label' is deprecated in type definitions. What you had before was fine which was to define 'name' here and 'label' in "types can be standardized" spec.